scripts/models/Video.js

```diff
@@ -0,0 +1,20 @@
+import { Media } from './Media.js';
+
+export class Video extends Media {
+  constructor(data) {
+    super(data);
+    this._id = data.id;
+    if (data.video !== undefined) {
+      this._video = data.video;
+    }
```

Comment:
No need to check if it's defined or not, as if `this._video` will remain undefined anyway if not passing the condition
```suggestion
    this._video = data.video;
```